internal/pkg/cli/deploy/env.go

```diff
@@ -150,108 +150,6 @@ func (d *envDeployer) Validate(mft *manifest.Environment) error {
 	return d.validateCDN(mft)
 }
 
-func (d *envDeployer) validateCDN(mft *manifest.Environment) error {
```

Comment:
There isn't any code change in `cli/deploy/env.go`. These are just moving codes around so that private methods are moved to the end of the file.

Reply:
I personally prefer to put all the methods that belong to one struct together and then sort them by public -> private, because they form a more consistent story (and consistent with most files in our codebase).

Reply:
All of the methods' receivers are the same in this file though...so my change here doesn't contradict with your prefernce 🤔

Reply:
oh gotcha sorry i misread